model_compression_toolkit/gptq/common/gptq_constants.py

```diff
@@ -8,4 +9,21 @@
 WEIGHTS_QUANTIZATION_PARAMS = 'weights_quantization_params'
 PTQ_MIN_RANGE = "_min_range"
 PTQ_MAX_RANGE = "_max_range"
```

Comment:
Because some are used with concatenation to a string variable, e.g., `layer.name + PTQ_MIN_RANGE`.
I'll changed that and add the "_" when using the variable instead of in the variable value, to keep it consistent.